schema-bridge.graphql

```diff
@@ -154,18 +304,54 @@ type UsageMetricsDailySnapshot @entity {
   " Protocol this snapshot is associated with "
   protocol: Protocol!
 
+  " Number of cumulative unique addresses that bridged funds or messages in and out of this chain "
+  cumulativeUniqueUsers: Int!
+
   " Number of unique daily active users "
   dailyActiveUsers: Int!
 
-  " Number of cumulative unique users "
-  cumulativeUniqueUsers: Int!
+  " Total number of unique addresses that provided liquidity to the bridge "
+  cumulativeUniqueLiquidityProviders: Int!
+
+  " Number of unique addresses that provided liquidity to the bridge in a day "
+  dailyActiveLiquidityProviders: Int!
+
+  " Total number of unique addresses that sent a message through the bridge "
+  cumulativeUniqueMessageSenders: Int!
+  
+  " Number of unique addresses that sent a message through the bridge in a day "
+  dailyActiveMessageSenders: Int!
 
   " Total number of transactions occurred in a day. Transactions include all entities that implement the Event interface. "
   dailyTransactionCount: Int!
 
+  " Total number of transactions transferring funds crosschain in a day "
+  dailyTransferCount: Int!
+
+  " Total number of deposits providing liquidity to a liquidity based pool in a day "
+  dailyDepositCount: Int!
+
+  " Total number of withdrawals from a liquidity based pool in a day "
+  dailyWithdrawCount: Int!
+
+  " Total number of messages sent through the bridge, originating in this chain in a day"
+  dailyMessageSentCount: Int!
+
   " Total number of pools "
   totalPoolCount: Int!
 
+  " We call a pool route each connection a token has with another chain. For example, if USDC can be bridged from the current chain to 2 others, it will have 2 pool routes. Not all assets can be bridged to all chains. "
+  totalPoolRouteCount: Int!
+
+  "  Number of routes where the destination asset is canonical (i.e. it is not wrapped)"
+  canonicalRouteCount: Int!
```

Comment:
I was confused if this would be total or daily, suggest renaming to `totalCanonicalRouteCount` (or `dailyCanonicalRouteCount`?)

same with `wrappedRouteCount` and `supportedTokenCount`

Reply:
`total` :P 
Renamed!